shared/helpers/view.js

```diff
@@ -67,7 +67,13 @@ function getClientPlaceholder(viewName, viewOptions) {
 
   // create a list of data attributes
   var attrString = _.inject(viewOptions, function(memo, value, key) {
-    if (_.isArray(value) || _.isObject(value)) { value = JSON.stringify(value); }
+    if (_.isArray(value) || _.isObject(value)) {
+      if (key === '_block' && value.contructor === Handlebars.SafeString) {
```

Comment:
Comparing the constructor is more precise in that it doesn't inspect the inheritance chain unless we want to know if an object is, or inherits from, a particular type. Probably not something to worry about in this case so I have no problem changing it if instance of is the desired form.